docs/books/user-guide/using-console.adoc

```diff
@@ -17,110 +17,100 @@ specific language governing permissions and limitations
 under the License
 ////
 
+[id='using-router-console']
 = Using {ConsoleName}
 
-[[console-overview]]
-== Console Overview
+{ConsoleName} is a web console for monitoring the status and performance of {RouterName} router networks.
 
-The console is an HTML based web site that displays information about a qpid dispatch router network.
+== Setting up access to the web console
 
-The console requires an HTML web server that can serve static html, javascript, style sheets, and images.
+Before you can access the web console, you must configure a `listener` to accept HTTP connections for the web console and serve the console files.
 
-The console only provides limited information about the clients that are attached to the router network and is therfore more appropriate for administrators needing to know the layout and health of the router network.
+.Procedure
 
-[[console-installation]]
-== Console Installation
+. On the router from which you want to access the web console, open the {RouterConfigFile} configuration file.
 
-[discrete]
-=== Prerequisites
-
-The following need to be installed before running a console:
-
-* One or more dispatch routers. See the documentation for the dispatch
-router for help in starting a router network.
-* A websockets to tcp proxy.
-* A web server. This can be any server capable of serving static
-html/js/css/image files.
-
-[discrete]
-=== Procedure
-
-To install a websockets to tcp proxy:
+. Add a `listener` to serve the console.
++
+--
+This example creates a `listener` that clients can use to access the web console:
 
+[options="nowrap",subs="+quotes"]
 ----
-sudo dnf install python-websockify
-websockify localhost:5673 localhost:5672
+listener {
+    host: 0.0.0.0
+    port: 8672
+    role: normal
+    http: true
+    httpRootDir: /usr/share/qpid-dispatch/console
+}
 ----
+`http`:: Set this attribute to `true` to specify that this `listener` should accept HTTP connections instead of plain AMQP connections.
```

Comment:
Is there anything to be said about the other attributes -- _host, port_, and _role_ -- or are these well understood by users?